scripts/update_plugin_repos.py

```
@@ -1,123 +1,38 @@
-    
-    print(f"\n✅ Updated {success_count}/{len(plugins)} plugins successfully!")
-    
-    if success_count < len(plugins):
-        print("⚠️  Some plugins failed to update. Check the errors above.")
+
+    if not (MONOREPO_DIR / ".git").exists():
+        print(f"Error: {MONOREPO_DIR} is not a git repository")
+        return 1
+
+    print(f"Updating {MONOREPO_DIR}...")
+    result = subprocess.run(
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)